app/views/students/shared/_attachments.html.erb

```diff
@@ -0,0 +1,10 @@
+<tr class="nested-fields">
+  <td data-for="type"><%= f.text_field :type, class: 'form-control form-control-sm' %></td>
+  <td data-for="filename"><%= f.file_field :attachment, multiple: true  %></td>
```

Comment:
How would the `multiple: true` change the behavior? I feel it would be better to upload 1 file per row.